src/shared/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "country-levels": "https://github.com/hyperknot/country-levels/releases/download/v2.0.1/export_medium.tgz",
```

Comment:
This cannot live in `src/shared`; it will push us over payload limits and slow everything down enormously. This needs to be factored out or away.